src/Fixer/Casing/NativeFunctionTypeDeclarationCasingFixer.php

```diff
@@ -42,7 +42,12 @@ final class NativeFunctionTypeDeclarationCasingFixer extends AbstractFixer
      * object   PHP 7.2
      * static   PHP 8.0 (return type only)
      * mixed    PHP 8.0
-     * never    PHP 8.1
+     * false    PHP 8.0 (union return type only)
```

Comment:
```suggestion
     * true     PHP 8.0 (union return type only)
     * false    PHP 8.0 (union return type only)
```

Reply:
is it really that 8.0 cares about `false` and `null`, but not `true`, @Wirone ?

Reply:
https://php.watch/versions/8.2/true-type
`true` as type will added in 8.2 (`false` was added in 8.0 but only for unions)

Reply:
@keradus it was already discussed [before](#discussion_r966467158) 😉 it's good how it is right now.